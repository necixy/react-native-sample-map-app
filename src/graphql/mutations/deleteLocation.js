import gql from "graphql-tag";
export default gql`
  mutation DeleteLocation($id: ID!) {
    deleteLocation(id: $id)
  }
`;
