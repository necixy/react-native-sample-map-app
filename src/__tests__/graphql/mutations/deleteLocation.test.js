import deleteLocation from "../../../graphql/mutations/deleteLocation";

describe("deleteLocation Mutation Testing", () => {
  it("deleteLocation Mutation Snapshots correctly", () => {
    expect(deleteLocation).toMatchSnapshot();
  });
});
