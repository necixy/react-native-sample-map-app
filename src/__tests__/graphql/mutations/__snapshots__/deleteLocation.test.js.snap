// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deleteLocation Mutation Testing deleteLocation Mutation Snapshots correctly 1`] = `
Object {
  "definitions": Array [
    Object {
      "directives": Array [],
      "kind": "OperationDefinition",
      "name": Object {
        "kind": "Name",
        "value": "DeleteLocation",
      },
      "operation": "mutation",
      "selectionSet": Object {
        "kind": "SelectionSet",
        "selections": Array [
          Object {
            "alias": undefined,
            "arguments": Array [
              Object {
                "kind": "Argument",
                "name": Object {
                  "kind": "Name",
                  "value": "id",
                },
                "value": Object {
                  "kind": "Variable",
                  "name": Object {
                    "kind": "Name",
                    "value": "id",
                  },
                },
              },
            ],
            "directives": Array [],
            "kind": "Field",
            "name": Object {
              "kind": "Name",
              "value": "deleteLocation",
            },
            "selectionSet": undefined,
          },
        ],
      },
      "variableDefinitions": Array [
        Object {
          "defaultValue": undefined,
          "directives": Array [],
          "kind": "VariableDefinition",
          "type": Object {
            "kind": "NonNullType",
            "type": Object {
              "kind": "NamedType",
              "name": Object {
                "kind": "Name",
                "value": "ID",
              },
            },
          },
          "variable": Object {
            "kind": "Variable",
            "name": Object {
              "kind": "Name",
              "value": "id",
            },
          },
        },
      ],
    },
  ],
  "kind": "Document",
  "loc": Object {
    "end": 71,
    "start": 0,
  },
}
`;
